packages/settings-library/src/settings_library/node_ports.py

```diff
@@ -0,0 +1,46 @@
+from typing import ClassVar, Final
+
+from models_library.basic_types import PortInt
+from pydantic import Field, NonNegativeInt, PositiveInt, SecretStr
+
+from ._constants import MINUTE
+from .base import BaseCustomSettings
+from .postgres import PostgresSettings
+from .storage import StorageSettings
+
+NODE_PORTS_400_REQUEST_TIMEOUT_ATTEMPTS_DEFAULT_VALUE: Final[NonNegativeInt] = 3
+
+
+class StorageAuthSettings(BaseCustomSettings):
+    NODE_PORTS_STORAGE_HOST: str
+    NODE_PORTS_STORAGE_PORT: PortInt
+
+    NODE_PORTS_STORAGE_LOGIN: str
+    NODE_PORTS_STORAGE_PASSWORD: SecretStr
+
+    @property
+    def base_url(self) -> str:
```

Comment:
Why not extending `StorageAuthSettings` from `StorageAuth` ?

Reply:
Good point, I was not thinking of using it like this initially. Will inherit